app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -2,7 +2,19 @@ class OmniauthCallbacksController < ApplicationController
   def callback
     auth = request.env["omniauth.auth"]
 
-    session[:user_info] = auth.extra.raw_info
+    session[:user_info] = if Rails.env.development?
```

Comment:
Maybe one for another day, but is this behaviour documented somewhere e.g. in the README?

Reply:
Sorry, that was meant to stay local. I will pull it back out

Reply:
Also, RE, the tests, I was making more changes now. Let me push them up too.

Reply:
👍